examples/scales/log_bar.py

```diff
@@ -3,26 +3,24 @@
 Log Bar
 =======
 
+Plotting a bar chart with a logarithmic y-axis.
 """
 import matplotlib.pyplot as plt
 import numpy as np
 
 data = ((3, 1000), (10, 3), (100, 30), (500, 800), (50, 1))
 
-plt.xlabel("FOO")
```

Comment:
Can we just change these to not 'FOO'?  It is good practice to always label your axes :smiling_imp: